lib/brcobranca/remessa/cnab400/credisis.rb

```diff
@@ -81,42 +92,95 @@ def monta_detalhe(pagamento, sequencial)
-          detalhe << Brcobranca::Util::Empresa.new(documento_cedente).tipo  # tipo de identificacao da empresa      9[02]
-          detalhe << documento_cedente.to_s.rjust(14, '0')                  # cpf/cnpj da empresa                   9[14]
+          detalhe << tipo_identificacao_cedente                             # tipo de identificacao da empresa      9[02]
+          detalhe << documento_cedente.to_s.rjust(14, '0')     # cpf/cnpj da empresa                   9[14]
           detalhe << agencia                                                # agencia                               9[04]
-          detalhe << ''.rjust(1, ' ')                                       # brancos                               X[01]
-          detalhe << conta_corrente                                         # conta corrente                        9[08]
+          detalhe << conta_corrente.rjust(8, '0')                           # conta corrente                        9[08]
           detalhe << digito_conta                                           # dac                                   9[01]
-          detalhe << ''.rjust(6, ' ')                                       # complemento do registro (brancos)     X[06]
-          detalhe << pagamento.documento_ou_numero.to_s.ljust(25) # identificacao do tit. na empresa      X[25]
-          detalhe << formata_nosso_numero(pagamento.nosso_numero.to_s)      # nosso numero                          9[11]
-          detalhe << ''.rjust(37, ' ')                                      # brancos                               X[37]
-          detalhe << pagamento.numero.to_s.rjust(10, '0') # numero do documento                   X[10]
-          detalhe << pagamento.data_vencimento.strftime('%d%m%y')           # data do vencimento                    A[06]
-          detalhe << pagamento.formata_valor                                # valor do documento                    9[13]
-          detalhe << ''.rjust(11, ' ')                                      # brancos                               X[11]
-          detalhe << pagamento.data_emissao.strftime('%d%m%y')              # data de emissao                       9[06]
-          detalhe << ''.rjust(4, ' ')                                       # brancos                               X[04]
-          detalhe << pagamento.formata_valor_mora(4).ljust(6, '0')          # valor mora ao dia                     9[06]
-          detalhe << pagamento.formata_percentual_multa.ljust(6, '0')       # valor multa                           9[06]
-          detalhe << ''.rjust(33, ' ')                                      # brancos                               X[33]
-          detalhe << pagamento.formata_valor_desconto                       # valor do desconto                     9[13]
+          detalhe << ' ' * 26                                               # brancos                               X[26]
```

Comment:
```suggestion
          detalhe << brancos(26)                                            # brancos                               X[26]
```
Acho que faria mais sentido do que `' ' * 26` 🤔

Reply:
🔝 tava procurando algo assim.

Reply:
Achei que esse método já existia, mas pensando bem, acho que é claro " " * x 🤔

Reply:
Não é o mesmo que `''.rjust(26, ' ')` que é feito nos métodos acima? Porque esse tem que ser diferente? 🤔

Reply:
Esquece, vi que removeu os rjust haha

Reply:
É a mesma coisa, antes usava `''.rjust(26, ' ')`, daí mudei pra usar *

Reply:
> Achei que esse método já existia, mas pensando bem, acho que é claro " " * x 🤔

Não, era uma sugestão mesmo hahahah